frontend/src/app/main/ui/workspace/tokens/style_dictionary.cljs

```diff
@@ -87,8 +87,34 @@
       (and (not has-references?) out-of-scope)
       {:errors [(wte/error-with-value :error.style-dictionary/invalid-token-value-opacity value)]}
 
-      (and has-references? out-of-scope)
-      (assoc parsed-value :warnings [(wtw/warning-with-value :warning.style-dictionary/invalid-referenced-token-value value)])
+      (and has-references? out-of-scope parsed-value)
+      (assoc parsed-value :warnings [(wtw/warning-with-value :warning.style-dictionary/invalid-referenced-token-value-opacity value)])
+
+      :else {:errors [(wte/error-with-value :error.style-dictionary/invalid-token-value value)]})))
+
+
+(defn- parse-sd-token-stroke-width-value
+  "Parses `value` of a dimensions `sd-token` into a map like `{:value 1 :unit \"px\"}`.
+  If the `value` is not parseable and/or has missing references returns a map with `:errors`.
+  If the `value` is parseable but is out of range returns a map with `warnings`."
+  [value has-references?]
+
+  (let [parsed-value (wtt/parse-token-value value)
+        out-of-scope (not (< 0 (:value parsed-value)))
```

Comment:
This gives error for value "0".

It should be `out-of-scope (< (:value parsed-value) 0)`